main/main.ino

```diff
@@ -1433,6 +1445,14 @@ void stateMeasures() {
   SYSdata["m5batpower"] = (float)M5.Axp.GetBatPower();
   SYSdata["m5batchargecurrent"] = (float)M5.Axp.GetBatChargeCurrent();
   SYSdata["m5apsvoltage"] = (float)M5.Axp.GetAPSVoltage();
+#  endif
+#  ifdef ZradioCC1101
+  SYSdata["mhz"] = (int)receiveMhz;
+#  endif
+#  if defined(ZgatewayRTL_433)
+  SYSdata["minimumRssi"] = (int)getMinimumRSSI();
+  SYSdata["currentRssi"] = (int)getCurrentRSSI();
+  SYSdata["messageCount"] = (int)getMessageCount();
```

Comment:
Could you reduce the length of the keys please and make it more specific to the CC1101 please

Reply:
Hummm, these only apply to the RTL package, so how about this ?

RTLminRssi
RTLRssi
RTLCnt